server/src/main/java/io/mantisrx/server/master/domain/DataFormatAdapter.java

```diff
@@ -519,10 +520,12 @@ public static IMantisStageMetadata convertMantisStageMetadataWriteableToMantisSt
 
     public static MantisJobMetadataWritable convertMantisJobMetadataToMantisJobMetadataWriteable(IMantisJobMetadata jobMetadata) {
 
+        long startedAt = (jobMetadata.getStartedAtInstant().isPresent()) ? jobMetadata.getStartedAtInstant().get().toEpochMilli() : -1;
```

Comment:
can use the optional APIs to something like `jobMetadata.getStartedAtInstant().orElse(-1)`. Also, might want to define the default `startedAt` as a static constant, seems it could be 0 or -1

Reply:
Good suggestion will add